README.md

```diff
@@ -1,5 +1,7 @@
 ![logo](https://raw.githubusercontent.com/bakkenbaeck/iOS-playbook/master/assets/logo-v1.png)
 
+Don't forget to check out our [suite of components and helper methods](http://github.com/usesweet), to avoid duplication of work reimplementing them yourself..
```

Comment:
Nice addition! I think the wording should be company agnostic, not say "our" but "our favorite" or something like that. Also might be better to place this at the bottom of the README instead of at the top.